src/main/java/com/wynntils/modules/utilities/managers/LevelingManager.java

```diff
@@ -41,6 +42,22 @@ public static void update() {
         XP_SET.releaseEntries();
     }
 
+    /**
+     * Get Excepted Grind Time for leveling up
```

Comment:
Excepted or expected?
```suggestion
     * Get expected grind time for leveling up
```